docs/sources/setup-grafana/configure-security/configure-authentication/keycloak/index.md

```diff
@@ -0,0 +1,139 @@
+---
+aliases:
+  - /docs/grafana/latest/auth/keycloak/
+  - /docs/grafana/latest/setup-grafana/configure-security/configure-authentication/keycloak/
+description: Keycloak Grafana OAuthentication Guide
+keywords:
+  - grafana
+  - keycloak
+  - configuration
+  - documentation
+  - oauth
+title: Configure Keycloak OAuth2 authentication
+weight: 200
+---
+
+# Configure Keycloak OAuth2 authentication
+
+Refer to [Generic OAuth authentication](../generic-oauth) for extra configuration options available for this provider.
```

Comment:
Another thought - would it be worth having a little blurb about what this article is about ("Keycloak OAuth2 authentication allows you to ..."), and then having a separate subsection for `##Configure Keycloak OAuth2 in Grafana` or something? Basically, similar to the format that other authN setup guides follow.